internal/core/model.rs

```diff
@@ -1117,18 +1117,35 @@ impl<C: RepeatedComponent + 'static> Repeater<C> {
 pub struct StandardListViewItem {
     /// The text content of the item
     pub text: crate::SharedString,
+
+    /// If set to `true` the text can be edited.
+    pub editable: bool,
 }
 
-impl From<&str> for StandardListViewItem {
```

Comment:
Is the removal of this possibly a breaking change?

Reply:
Yes, that looks like a breaking change.

Reply:
It could be stay. What I can to is set on `From<&str>` `editable` to `false`.